exercises/affine-cipher/src/test/java/AffineCipherTest.java

```diff
@@ -0,0 +1,136 @@
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public class AffineCipherTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void testEncodeYes() {
+        AffineCipher affineCipher = new AffineCipher();
```

Comment:
From what I can tell this is the same for all tests? So it might be nice to pull it out into a `@Before` block? 🙂